contracts/MedianOracle.sol

```diff
@@ -0,0 +1,223 @@
+pragma solidity 0.4.24;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
```

Comment:
I wonder if you can specify the version explicitly as i've done here .. 

https://github.com/ampleforth/cross-chain-ample/pull/60/files#diff-ff4eb0f76489225a40c20425b35dd4ae531717d0f53736363ad7b1bf62b681b6R4

Reply:
so I tried a few things here. I could `yarn add` from the github path as you had, in the case "https://github.com/OpenZeppelin/openzeppelin-**solidity**.git#2.0.0" actually resolves to "https://github.com/OpenZeppelin/openzeppelin-**contracts**.git#2.0.0", but the package name remains "openzeppelin-solidity" for purposes of importing from the contract code